erdos/src/dataflow/operators/filter.rs

```diff
@@ -93,11 +94,17 @@ where
     {
         let op_name = format!("FilterOp_{}", self.id());
 
-        crate::connect_one_in_one_out(
+        let write_stream = OperatorStream::new(
+            &format!("{}-write-stream", op_name),
+            Arc::clone(&self.get_graph()),
+        );
```

Comment:
Extremely minor OCD nit: In the rest of the LINQ operators, we have an empty line after `let write_stream` instead of before.